### PR TITLE
Big clean up !

### PR DESCRIPTION
**Pull request by GoOz**
Don't really expect a thorough review but really keeping you posted instead to what has been done so far. So here it is:
- [x] Removed mixins `padded` and `spaced`
- [x] Added buttons global classes
- [x] Ensure retro-compatibility
- [x] Removed Lato font from UI & moved to the stack (you'll need to add a `<link>` tag to the index.ejs of your app `<link rel="stylesheet" type="text/css" href="//{{.Domain}}/assets/fonts/fonts.css">`
- [x] Fix modal position on desktop
- [x] Added Nav component
- [x] Clean up comments

### CODE REVIEW

File: stylus/components/alerts.styl [hunk 1]
```diff
@@ -19,7 +19,7 @@ $alerts
         box-sizing        border-box
         box-shadow        0 em(6px) em(18px) 0 rgba(50, 54, 63, .23)
         background-color  emerald
-        padded            .8em, 1em
+        padding            .8em, 1em
```
Comment (nono) [inline stylus/components/alerts.styl]: Alignment for the right column is broken

File: stylus/components/button.styl [hunk 2]
```diff
@@ -46,7 +67,8 @@ $button
             top        em(-1px)
 
 
-$button--regular
+$button--regular // depreciated
```
Comment (nono) [inline stylus/components/button.styl]: s/depreciated/deprecated/

File: stylus/components/alerts.styl (hunk 1 above)
Comment (GoOz) [reply stylus/components/alerts.styl]: Good catch :)

File: stylus/components/button.styl (hunk 2 above)
Comment (GoOz) [reply stylus/components/button.styl]: Oh man, it sounded like depreciated in my head so I wrote it that way, thanks.

**Pull request merged**